games-emulation/openmsx/openmsx-16.0.ebuild

```diff
@@ -0,0 +1,112 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python3_{6,7,8,9} )
+
+inherit desktop flag-o-matic python-any-r1 readme.gentoo-r1 toolchain-funcs xdg
+
+DESCRIPTION="MSX emulator that aims for perfection"
+HOMEPAGE="https://openmsx.org/"
+SRC_URI="https://github.com/openMSX/openMSX/releases/download/RELEASE_${PV//./_}/${P}.tar.gz"
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+
+RDEPEND="dev-lang/tcl:0=
+	dev-libs/libxml2
+	media-libs/alsa-lib
+	media-libs/libogg
+	media-libs/libpng:0=
+	media-libs/libsdl2[sound,video]
+	media-libs/libtheora
+	media-libs/libvorbis
+	media-libs/sdl2-image[png]
+	media-libs/sdl2-ttf
+	>=media-libs/glew-1.3.2:0=
+	virtual/opengl"
```

Comment:
Sorry, I was doing one more final check and noticed through `readelf -d` that the dependencies are outdated.

* libxml2 was dropped in 2014.
* SDL_image was dropped in 2010!!
* It should now depend on zlib.